.github/workflows/ci.yml

```diff
@@ -60,7 +60,7 @@ jobs:
     strategy:
       matrix:
         python-version: ["3.8", "3.11"]
-        os: ubuntu-latest
+        os: [ubuntu-latest]
         include:
           - os: ubuntu-20.04
             python-version: 3.6
```

Comment:
Nitpick but can you quote "3.6" for consistency ;)

Reply:
fair